app/javascript/controllers/selection_controller.js

```diff
@@ -0,0 +1,38 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["rowSelection"];
+
+  connect() {
+    this.localStorageKey =
+      location.protocol + "//" + location.host + location.pathname;
```

Comment:
I thought I was asked to use the url as the key. This opens it up to anything. What if we pass a value, but make the url the default? Please see 4efdfbe9d81208f524dc614f679420a15379bffb.

Reply:
Having a default value is a fantastic idea :) The only one thing I would ask is that you make a LookBook preview showing how to use the non-default value.  I know visually it won't change anything in the preview, but it could help someone using this controller know what is going on.

Reply:
I tried. Please see 62ea06896306b4be73affc9b5cfcb161c667ddc7.

Reply:
Does what I needed it to do, thank you.